packages/story-editor/tsconfig.json

```
@@ -4,9 +4,17 @@
     "rootDir": "src",
     "declarationDir": "dist-types"
   },
+  "references": [
+    { "path": "../../packages/react" },
+    { "path": "../../packages/elements" }
+  ],
   "include": [
     "src/app/config/*",
     "src/types",
-    "src/app/api/*"
+    "src/app/taxonomy/**/*",
+    "src/app/story/**/*",
+    "src/app/history/**/*",
+    "src/app/api/*",
+    "src/utils/*"
```

The taxonomy provider only uses the `cleanForSlug` util, so I think you want to just include that one.

```suggestion
    "src/utils/cleanForSlug"
```

Good opportunity to convert it to TS by the way :-)